libraries/botbuilder-middleware-watson-nlu/example/botbuilder-emotion-middleware-testbot.bot

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I would remove the .bot files, since they are deprecated.

Reply:
Done.